test/image_test.py

```diff
@@ -64,10 +65,13 @@ def get_all_dockerfile_commands(image_id: str, servicer) -> str:
 
 @pytest.fixture(params=get_args(ImageBuilderVersion))
 def builder_version(request, server_url_env, modal_config):
-    version = request.param
+    builder_version = request.param
+    python_version = f"{0}.{1}".format(sys.version_info)
+    if python_version not in SUPPORTED_PYTHON_SERIES[builder_version]:
+        pytest.skip("Python version not supported by Image Builder version")
```

Comment:
This produces a lot of skip tests — maybe there is a cleaner way.

Reply:
Yeah I'm not sure. Wasn't even aware you could parameterize fixtures like this
One idea is that you create a reversed mapping of SUPPORTED_PYTHON_SERIES which maps from python version to supported image builder versions, and send that in to params instead? Then it will only be parameterized on the supported image builder versions?

Reply:
Nice — yeah that's better.